### PR TITLE
Update in Installation Notes

### PR DESCRIPTION
**Pull request by joinimran**
These updates can be delivered via Windows Updates.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6322

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/update/servicing-stack-updates.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Typically, the improvements are reliability and performance improvements that do
 * Servicing stack updates contain the full servicing stack; as a result, typically administrators only need to install the latest servicing stack update for the operating system.
 * Installing servicing stack update does not require restarting the device, so installation should not be disruptive. 
 * Servicing stack update releases are specific to the operating system version (build number), much like quality updates.
-* Search to install latest available [Servicing stack update for Windows 10](https://portal.msrc.microsoft.com/en-us/security-guidance/advisory/ADV990001).
+* It can be delivered via Windows updates or search to install latest available [Servicing stack update for Windows 10](https://portal.msrc.microsoft.com/en-us/security-guidance/advisory/ADV990001).
```
Comment (illfated) [inline windows/deployment/update/servicing-stack-updates.md:56]:
```suggestion
* Servicing stack updates can be delivered via Windows Update, or you can perform a search to install the latest available [Servicing stack update for Windows 10](https://portal.msrc.microsoft.com/en-us/security-guidance/advisory/ADV990001).
```
This suggestion may need review and possible changes, so I am looking forward to feedback from @JohanFreelancer9 for confirmation.

Comment (JohanFreelancer9) [reply windows/deployment/update/servicing-stack-updates.md:56]: Looks good. Please just remove the locale from the link.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/deployment/update/servicing-stack-updates.md (hunk 1 above)
Comment (illfated) [reply windows/deployment/update/servicing-stack-updates.md:56]: Thank you for the reminder. Done.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me. Thank you for creating the PR and applying the suggested changes.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**